scripts/deps.sh

```diff
@@ -126,5 +131,10 @@ fi
 msg "Installing pinned Rust toolchain and components"
 rustup show
 
+if [[ $RUSTUP_NEW_INSTALL = 1 ]]; then
+  msg "rustup was just installed. Ensure cargo is on the PATH with:"
```

Comment:
Maybe I should colorize this output as well. People will probably *still* ignore it because it's white text that isn't the last line of output. Yellow should be good?